roles/openshift_grafana/defaults/main.yaml

```diff
@@ -0,0 +1,54 @@
+---
+
+openshift_grafana_state: present
+openshift_grafana_namespace: grafana
+openshift_grafana_pod_timeout: 300
+openshift_grafana_prometheus_namespace: "{{ openshift_prometheus_namespace | default('openshift-metrics') }}"
+openshift_grafana_prometheus_serviceaccount: prometheus
+openshift_grafana_serviceaccount_name: grafana
+openshift_grafana_serviceaccount_annotations: []
+l_openshift_grafana_serviceaccount_annotations:
+  - serviceaccounts.openshift.io/oauth-redirectreference.primary='{"kind":"OAuthRedirectReference","apiVersion":"v1","reference":{"kind":"Route","name":"grafana"}}'
+openshift_grafana_datasource_name: "prometheus"
+openshift_grafana_node_exporter: false
+openshift_grafana_graph_granularity: "2m"
+openshift_grafana_node_selector: {"region":"infra"}
```

Comment:
Can we use any node as default ?!
In case we can't use the infra nodes it will try any region and master as well.

e.g
openshift_grafana_node_selector: ''

Reply:
It follows standard our playbooks pattern. Either way, its a good practice to provide the component label. I would not want to default to `empty selector`. Additionally running metrics/logging infrastructure together with app workloads is not a good design pattern. 

Happy to be challenged on this one :)